CHANGELOG.md

```diff
@@ -0,0 +1,314 @@
+# Fastlane CHANGELOG
+
```

Comment:
Indeed. Do you usually leave this `## Master` section empty at start of a new release, or put something like `_nothing yet_` there until the first PR adds an entry?